client/src/main/scala/drt/client/components/MultiDayExportComponent.scala

```diff
@@ -137,8 +137,10 @@ object MultiDayExportComponent extends WithScalaCssImplicits {
 
                       if (props.loggedInUser.hasRole(ArrivalsAndSplitsView)) {
                         val exports = props.portCode match {
-                          case PortCode("LHR") if LhrRedListDatesImpl.dayHasPaxDiversions(SDate(state.endDate.date)) =>
-                            List(ExportArrivalsWithRedListDiversions("Reflect pax diversions"), ExportArrivalsWithoutRedListDiversions("Don't reflect pax diversions"))
+                          case PortCode("LHR") if (isRedListActive(SDate(state.startDate.date)) || isRedListActive(SDate(state.endDate.date)))
```

Comment:
This logic isn't right. If the start date is before the red list and the end date is after the red list then we should still ask the question